appveyor.yml

```diff
@@ -15,12 +15,14 @@ environment:
 install:
   # We need wheel installed to build wheels
   - "git submodule update --init --recursive"
-  - "%PYTHON%\\python.exe -m pip install wheel"
+  - "%PYTHON%\\python.exe -m pip install -U wheel"
```

Comment:
At all? Oh :/
I just wanted to be sure we had a recent enough (can't remember the exact version) so that pip knows how to use it for caching.